content/docs/tools-ui/ui-dev-messages.md

```diff
@@ -0,0 +1,537 @@
+
+![O3DE Standard Information Icon](/images/tools-ui/overview/standard-icons/information.png)
+
+Indicates additional (non-critical) information that doesn't require action from the user. This may be used to inform the user that something is ready to view, such as for system feedback or to signify changes since the last interaction.
+
+*Used for: information, guidance, exceptions*
+
+
+## Message Colors
+
+In messages, semantic colors indicate the purpose of a message, which users can quickly recognize by the color. For instance, the color green has positive connotations, so it's used to communicate success or confirmation.
+
+- Error: `#FA2727`
+- Warning: `#FFAA22`
+- Success: `#58BC61`
+- Information: `#1E70EB`
+- Information icon: `#FFFFFF`
+
+![O3DE Message Colors](/images/tools-ui/overview/colors.png)
+
+* * *
+
+## Toasts
+
+In O3DE, *toast messages* are non-disruptive notifications. They can be displayed in the middle of a user's workflow without hindering the user's actions. They should appear over page content or above most elements on the bottom-right corner of the window.
+
+The following example demonstrates a toast that appears at the bottom-right corner of the **O3DE Animation Editor** window.
+![Example of a floating toast in the O3DE Animation Editor](/images/tools-ui/toasts/floating-toast-message-in-animation-editor.png)
+
+### Using standard icons
+Toasts convey the message's intent and must correspond with an appropriate icon to provide a consistent experience for the users. The following image shows toasts of different message intents: error/failure, warning, success, and information. 
+
+![Floating Toasts - Messages](/images/tools-ui/toasts/floating-toasts-messages.png)
+
+
+Refer to the following table to determine what icon to use in your toast, depending on the message's use case. 
+
+![Floating Toasts - Decision Table](/images/tools-ui/toasts/floating-toasts-decision-table.png)
+
+### General specification for toasts
+
+* Toasts are passive and non-disruptive messages. They shouldn't hinder the user's workflow.
+* Toasts must disappear after three seconds (or a maximum of five seconds). An exception is if the message includes a link that requires the user to take action for the toast to disappear.
+* Multiple toasts should not be stacked horizontally or vertically, as this may block the user's workflow.
+* The icon within a toast must be a standard icon: error/failure, warning, success, or information icon.
+* The toast must have a fixed width and should not expand to fit the content area.
+* Toasts must not contain a call-to-action button.
+* Toasts must appear above page content and on a screen where overlay layers are appropriate.
+* Toasts must not be used within a modal. Consider using them within larger systems like **O3DE Editor**, **Material Editor**, **Viewport**, and so on.
+
+
+![Floating Toast Messages - Best Practices](/images/tools-ui/toasts/floating-toast-messages-best-practices.png)
+
```

Comment:
Previous text:
- [Line 103-109]
   > #### General specification for Toasts
   >
   >   * Toasts must disappear after three seconds (or a maximum of five seconds), unless the message includes a link wherein the toast can persist on the screen until action is taken.
   >   * Multiple toasts should not be stacked, as this might block any user action.
   >   * Keep the messages clear and concise, a maximum of two lines.
   >   * The icon within a toast must be a standard icon: error/failure, warning, success, or information icon.
   >   * It must not contain a call-to-action button.
   >   * The toast should appear above page content, and on a screen where overlay layers are appropriate.
   >   * This toast must not be used within a modal, consider using it within larger systems like Animation Editor, Material Editor, Viewport, etc.

- [Line 122-128]
   > #### Best practices for Toasts
   > 
   >   * Toasts are passive in nature, and should not hinder the user's workflow
   >   * Keep the messages clear and concise.
   >   * The toast must disappear after five seconds, unless the message contains a link.
   >   * The toast must have a fixed width, and should not expand to fit the content area.
   >   * It should not cover any page content, and multiple toasts must not be stacked horizontally.

Comments: 
   - There is some overlap in the general specs and best practices. I recommend dividing it so that general specs contains guidelines that users *must* follow and best practices contains recommendations that users are *suggested* to follow. 
   - Reworded "Multiple toasts should not be stacked, as this might block any user action." to specify to avoid stacking in a *horizontal* and *vertical* manner. I also assumed and specified that "user action" refers to the user's workflow before the toast appeared, and not to a link on the toast that requires the user to take action.